### PR TITLE
Responsive Mobile menu

### PR DESCRIPTION
**Pull request by TekyRey**
### **Responsive Mobile menu**

In this project we:

- Used JavaScript `classList` method to hide and show the mobile `nav `links
- styled the `mobile menu`
- Added `links `to redirect to respective sections on the web page

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -6,32 +6,20 @@ env:
...
-        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-  webhint:
-    name: Webhint
-    runs-on: ubuntu-18.04
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-node@v1
-        with:
-          node-version: "12.x"
-      - name: Setup Webhint
```
Comment (muneebulrehman) [inline .github/workflows/linters.yml]: Kindly add the web hint linter back, since there is no linter checking for your HTML.

File: index.html [hunk 2]
```diff
@@ -11,26 +11,27 @@
...
       rel="stylesheet"
     />
-    <title>Portfolio</title>
+    <link rel="shortcut icon" href="images/logo.png" />
+    <title>Rehema's Portfolio</title>
   </head>
   <body>
     <div class="page-content">
       <header class="header">
         <nav class="nav">
```
Comment (muneebulrehman) [inline index.html:20]: Your navbar is showing the hamburger icon on bigger screens. It should only be visible on the smaller screens and not on the bigger screens.
Here is a reference screenshot of your app.
image

- Optional. 
Your header layout is not matching the Figma template of your choosing. Kindly fix it.

File: index.html [hunk 3]
```diff
@@ -138,9 +139,9 @@ <h1>
...
             <div class="experience-section">
               <p class="canopy">Canopy</p>
-              <img src="images/Counter.svg" alt="" aria-hidden="true" />
+              <img src="images/Counter.svg" alt="Counter Image" aria-hidden="true" />
               <p class="bullet">Back End Dev</p>
-              <img src="images/Counter.svg" alt="" aria-hidden="true" />
+              <img src="images/Counter.svg" alt="Counter Image" aria-hidden="true" />
               <p class="bullet">2015</p>
             </div>
             <div>
```
Comment (muneebulrehman) [inline index.html:152]: - Optional
The images in your work cards are breaking. Each card is taking the whole screen and then some. Kindly fix it. The images are so big that I can't even take a screenshot. 😊

File: index.html [hunk 4]
```diff
@@ -365,33 +366,33 @@ <h3>Lets Connect</h3>
             <li class="languages">
               <div class="languages-title">
                 <h2>Languages</h2>
-                <img src="images/downarrow.svg" alt="" />
+                <img src="images/downarrow.svg" alt="down arrow icon" />
```
Comment (muneebulrehman) [inline index.html:374]: - Optional
Your language layout is also breaking. Consider fixing it.
image

File: index.html [hunk 5]
```diff
@@ -411,5 +412,6 @@ <h2>Contact me</h2>
         </div>
```
Comment (muneebulrehman) [inline index.html:417]: - Optional.
Your contact-me section is also breaking. Consider fixing it.
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @TekyRey , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- **_There are a lot of issues in your project, however since they were not in this Pr's requirements I have marked them as optional. However, I strongly recommend you try to fix them._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (TekyRey) [reply .github/workflows/linters.yml]: i get the error below after adding the `webhint linter` what could the problem be?
image

File: index.html (hunk 2 above)
Comment (TekyRey) [reply index.html:20]: Hi @muneebulrehman I don't get why this is what you are seeing from your side, the app looks okay from my side, I have tested it on both chrome and edge and this is how it looks. 

image
image
image
image
image
image

File: index.html [hunk 6]
```diff
@@ -11,26 +10,28 @@
...
+    <link rel="shortcut icon" href="images/logo.png" />
+    <title>Rehema's Portfolio</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
   </head>
   <body>
     <div class="page-content">
       <header class="header">
         <nav class="nav">
           <div class="mobile-menu">
             <button type="button">
```
Comment (youmari) [inline index.html:22]: - [ ] please make sure that in the 1447px wide screen the hamburger menu does not show up

File: index.html [hunk 7]
```diff
@@ -11,26 +10,28 @@
...
-              <img class="close-image" src="images/x.png" alt="" />
+              <img class="close-image" src="images/x.png" alt="close-icon" />
             </button>
             <div class="mobile-items">
               <a href="#portfolio">Portfolio</a>
               <a href="#about">About</a>
               <a href="#contact">Contact</a>
             </div>
```
Comment (youmari) [inline index.html:29]: - [ ] [OPTIONAL] please it would be better if you add the blur to your mobile menu in order to match the Design

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @TekyRey , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ No linter error 
✔️ Great title and short summary

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — APPROVED** [review]
### Hi @TekyRey,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**